trussworks/react-uswds
# fix: Project's example doesn't seem to compile/run

# Summary

Fixes an issue where the example app wouldn't build. This was due to some breaking changes introduced in `react-router-dom` v6.

## Related Issues or PRs

closes #1922

## How To Test

navigate to `/example`
`yarn install && yarn build`
should not break or return errors

### Screenshots (optional)
